frontend/src/components/RecipeComments/RecipeComments.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import map from 'lodash/map';
+import { createComment } from '../../lib/apiClient';
+import './RecipeComments.css';
+
+const RecipeComments = ({ comments, recipeId, user }) => {
+  const [comment, setComment] = useState('');
+  const handleChange = (event) => {
+    event.preventDefault();
+    setComment(event.target.value);
+  };
+
+  const handleSubmit = () => {
+    createComment(comment, recipeId);
+  };
+
+  const renderForm = () => (
+    <form onSubmit={handleSubmit}>
+      <textarea id="comment-box" type="text" value={comment} placeholder="Type a comment" onChange={handleChange} />
+      <input type="submit" />
+    </form>
+  );
+
+  const renderComments = () => {
+    const commentsArr = [];
+    for (let i = comments.length - 1; i >= 0; i - 1) {
+      const commentArr = [];
+      const commenter = (`${comments[i].user.first_name} ${comments[i].user.last_name}`);
+      commentArr.push(commenter);
+      commentArr.push(comments[i].body);
+      commentsArr.push(commentArr);
+    }
+    return (
+      <div>
+        {map(commentsArr, value => (
+          <div className="comment-content">
+            <hr />
+            <p><strong>{value[0]}</strong></p>
+            <p>{value[1]}</p>
```

Comment:
Instead of doing the loop above, why not just map over `comments` and use `${value.user.first_name} ${value.user.last_name}` and `value.body` here?

Reply:
Originally, I had tried to map over the comments array directly, but I got some errors because the .map method from vanilla JS didn't like that there were objects nested inside the array. So I used the for loop to extract the necessary data and put it into arrays inside an array.

Reply:
Hmm, I'm not sure what that would be. I think mapping over comments directly would be preferred here, and maybe there was some other bug causing problems with map.